staging/src/k8s.io/legacy-cloud-providers/gce/gce.go

```diff
@@ -672,11 +698,13 @@ func (g *Cloud) NetworkURL() string {
 
 // SubnetworkURL returns the subnetwork url
 func (g *Cloud) SubnetworkURL() string {
+	g.initializer.Do(g.initializeSubnetworkURLAndIsLegacyNetwork)
```

Comment:
IIUC the code before the change, it is possible to provide subnetworkURL as part of config. In that case the logic wasn't called.

After the change, you will start doing additional GCE API calls.